kernel_hardening_checker/config_files/distros/get_kconfigs.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Links with raw content
+links=(
+    "https://raw.githubusercontent.com/oracle/kconfigs/refs/heads/gh-pages/out/Android 14 (6.1) aarch64/config"
+    "https://raw.githubusercontent.com/oracle/kconfigs/refs/heads/gh-pages/out/Debian 13 Trixie aarch64/config"
+    "https://raw.githubusercontent.com/oracle/kconfigs/refs/heads/gh-pages/out/Debian 13 Trixie x86_64/config"
+    "https://raw.githubusercontent.com/oracle/kconfigs/refs/heads/gh-pages/out/CentOS Hyperscale 9 aarch64/config"
+    "https://raw.githubusercontent.com/oracle/kconfigs/refs/heads/gh-pages/out/CentOS Hyperscale 9 x86_64/config"
+    "https://raw.githubusercontent.com/oracle/kconfigs/refs/heads/gh-pages/out/Fedora 39 Core aarch64/config"
+    "https://raw.githubusercontent.com/oracle/kconfigs/refs/heads/gh-pages/out/Fedora 39 Core x86_64/config"
+    "https://raw.githubusercontent.com/oracle/kconfigs/refs/heads/gh-pages/out/Oracle Linux 9 (UEK-NEXT) aarch64/config"
+    "https://raw.githubusercontent.com/oracle/kconfigs/refs/heads/gh-pages/out/Oracle Linux 9 (UEK-NEXT) x86_64/config"
```

Comment:
Could you please also rebase this branch onto the fresh master?
Thanks!

Reply:
Which Arch?) I believe there may be five possible kernels:

https://wiki.archlinux.org/title/Kernel

I've noticed that there are no links for aarm64, which seems a little unusual.

For the time being, I've added the stable and hardened one, as it was added before.

https://github.com/a13xp0p0v/kernel-hardening-checker/pull/165/commits/5399a9ee93862b504933b499c3613315eb095325

https://github.com/a13xp0p0v/kernel-hardening-checker/pull/165/commits/b441d4cba51811197e84bc50873c834e3ae58d13

If it would be helpful, the version could also be specified. For now, I have chosen the upstream one.:
![image](https://github.com/user-attachments/assets/0ecbc8cf-2feb-434f-9d78-85f21a68b3b1)